src/workerd/io/io-gate.h

```diff
@@ -51,10 +51,10 @@ class InputGate {
     virtual void inputGateWaiterAdded() {}
     virtual void inputGateWaiterRemoved() {}
 
-    static Hooks DEFAULT;
+    static const Hooks DEFAULT;
   };
 
-  InputGate(Hooks& hooks = Hooks::DEFAULT);
+  InputGate(Hooks& hooks = const_cast<Hooks&>(Hooks::DEFAULT));
```

Comment:
Generally any `const_cast` should come with a comment saying why it's OK. In this case: "const_cast OK because the object has no members, it's just a vtable."